### DISCUSSION TITLE
The right way to set up accounts and repositories

### DISCUSSION THREAD

**Discussion by LanguageDescriptions** [community/community#22610] [New to GitHub]
<p>I signed up for Github not knowing whether it would be useful to me, so I picked an unfortunately specific username (“Logoori”, which is the name of a language) that pertains to just one project (on that language). Now that I understand a bit more what one can do with Github, I’d like to be able to put multiple projects on Github. I can create a new repository under Logoori, for the Saami (language) repository, but /Logoori/Saami… is really misleading, because it implies that Saami is somehow hierarchically related to Logoori (it is not). I thought of creating separate accounts for each language project, but that’s blocked (using a single email address: I don’t want to proliferate email addresses).</p>
<p>I <em>think</em> that the way to conceptualize this is that I need a username and account that reflects the nature of all of these projects, such as “LanguageProjects”, then create repositories under that account. The horse is out of the barn, but maybe it’s not a huge problem: can a username be changed, so that all of these repositories will be under “LanguageProjects”? I prefer not to use my own name since I don;t intend to be the only one contributing to these reopsitories. If not, how <em>should</em> I be conceptualizing usernames, accounts, and repositories on Github? Hope you understand what I’m lookng for… mostly policy, and a bit of tech.</p>

**Comment by Wabri** [comment, marked as answer]
<p>I think you can do multiple choices.</p>
<p>First one is to change your account name, to do that you have to go to settings (up in the right of your github homepage):</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/5/52d249578a88d9b6a0c97f1979c20531675f2ad4.png" data-download-href="/uploads/short-url/bOFGJHqtPDrAdJGPeN21kv0ZGa8.png?dl=1" title="com1m.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/12409541/181072177-cf2ea069-d6a3-413a-8bf2-56069b15f31d.png" alt="com1m.png" data-base62-sha1="bOFGJHqtPDrAdJGPeN21kv0ZGa8" width="333" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/5/52d249578a88d9b6a0c97f1979c20531675f2ad4_2_333x500.png, https://aws1.discourse-cdn.com/github/original/2X/5/52d249578a88d9b6a0c97f1979c20531675f2ad4.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/5/52d249578a88d9b6a0c97f1979c20531675f2ad4.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/5/52d249578a88d9b6a0c97f1979c20531675f2ad4_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">com1m.png</span><span class="informations">354×530 17.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Than account (second option of the list):</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/c/cfb1f8a8634c5fcb45e9d6c75a80f0e3dca82d3a.png" data-download-href="/uploads/short-url/tDm8z9g9u8M3Dcr6VYmMO7jLX3Y.png?dl=1" title="com2m.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/12409541/181072178-e16b3811-5066-40da-b06b-bbf97f8156a8.png" alt="com2m.png" data-base62-sha1="tDm8z9g9u8M3Dcr6VYmMO7jLX3Y" width="321" height="500" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/c/cfb1f8a8634c5fcb45e9d6c75a80f0e3dca82d3a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">com2m.png</span><span class="informations">321×820 20.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Now you can see the botton “Change username”:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/e/e1f8e87a9b7b681d041452b57fb8e427ef58e1e2.png" data-download-href="/uploads/short-url/wf2IgLPL4plGlL9RD8Wbqqo6LBw.png?dl=1" title="com3m.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/12409541/181072179-381f1ca4-4dc4-496d-9d9d-849247590677.png" alt="com3m.png" data-base62-sha1="wf2IgLPL4plGlL9RD8Wbqqo6LBw" width="434" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/e/e1f8e87a9b7b681d041452b57fb8e427ef58e1e2_2_434x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/e/e1f8e87a9b7b681d041452b57fb8e427ef58e1e2_2_651x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/e/e1f8e87a9b7b681d041452b57fb8e427ef58e1e2.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/e/e1f8e87a9b7b681d041452b57fb8e427ef58e1e2_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">com3m.png</span><span class="informations">660×759 31.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Read and make your choice.</p>
<p>Second one is to create an organization once per language, but maybe is not worth it. To do that, you have to create a new organization:</p>
<p><img src="https://user-images.githubusercontent.com/12409541/181072180-acd8b8ec-4daf-43cf-88d2-07c202831052.png" alt="com4m.png" data-base62-sha1="8HFrM0pR0pwxf0LogDzBzTX1WXn" width="383" height="266"></p>
<p>You have to compile a form, but is not so difficult.</p>
<p>Make your choice.</p>
<p>If you have anything else to ask, we’re here.</p>
<p>Gabriele</p>

**Comment by LanguageDescriptions** [comment]
<p>After doing that, I messed everything up, and it took only a half-dozen variations to get control of the web pages, maybe net 30 minutes overhead, and now I have a more general system. Thanks.</p>
